src/helpers/script_helpers.sh

```diff
@@ -15,6 +15,7 @@ function command_steps {
   echo_if_all_true "stash_open_changes" "$STASH_OPEN_CHANGES" "$INITIAL_OPEN_CHANGES"
   steps
   echo_if_all_true "restore_open_changes" "$STASH_OPEN_CHANGES" "$INITIAL_OPEN_CHANGES"
+  echo_if_all_true "set_previous_branch \$(expected_previous_branch)"
```

Comment:
I think this step has to come before we restore the open changes on line 17. Otherwise changing branches with open changes that conflict with the previous branch breaks here. We really have to implement #176 to catch issues like this.